infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAssetDao.kt

```diff
@@ -24,25 +23,26 @@ import fi.fta.geoviite.infra.util.getDaoResponse
 import fi.fta.geoviite.infra.util.getInstant
 import fi.fta.geoviite.infra.util.getInstantOrNull
 import fi.fta.geoviite.infra.util.getIntId
-import fi.fta.geoviite.infra.util.getRowVersion
+import fi.fta.geoviite.infra.util.getLayoutRowVersion
 import fi.fta.geoviite.infra.util.idOrIdsEqualSqlFragment
 import fi.fta.geoviite.infra.util.queryOne
 import fi.fta.geoviite.infra.util.queryOptional
+import fi.fta.geoviite.infra.util.requireOne
 import fi.fta.geoviite.infra.util.setUser
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
 import org.springframework.transaction.annotation.Propagation
 import org.springframework.transaction.annotation.Transactional
 import java.sql.Timestamp
 import java.time.Instant
 
-data class DaoResponse<T>(val id: IntId<T>, val rowVersion: RowVersion<T>)
+data class DaoResponse<T>(val id: IntId<T>, val rowVersion: LayoutRowVersion<T>)
```

Comment:
Mietin kannattaisiko tämä nimetä nyt `LayoutDaoResponse`:ksi, sillä nimestä saa sen käsityksen että se voisi olla geneerisempikin (ts. käytössä mille tahansa versioidulle kamalle.) Toki paketointi `tracklayout`:in alle jo itsessään kertoo jo vähän siitä että tätä ei kuuluisi käyttää ihan mistä tahansa, mutta kumminkin

Reply:
Joo totta, tähänkin voisi ottaa layoutin mukaan kun se kerran tuli layout-kohtaiseksi.